SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  SKYChatConversationViewController.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import JSQMessagesViewController
+
+class SKYChatConversationViewController: JSQMessagesViewController {
+    public var skygear: SKYContainer = SKYContainer.default()
+    public var conversation: SKYConversation?
+    public var participants: [String: SKYRecord] = [:]
+    public var messages: [SKYMessage] = []
+
+    static let errorDomain: String = "SKYChatConversationViewControllerErrorDomain"
+    var errorCreator: SKYErrorCreator {
+        return SKYErrorCreator(defaultErrorDomain: SKYChatConversationViewController.errorDomain)
+    }
+}
+
+// MARK: - Initializing
+
+extension SKYChatConversationViewController {
+
+    public class func create() -> SKYChatConversationViewController {
+        return SKYChatConversationViewController(nibName: "JSQMessagesViewController",
```

Comment:
Oh, ok! I also see `SKYChatConversationListViewController` and `SKYChatParticipantListViewController` using this convention, then I should better follow too...

I personally think repeated init is much better, 3 lines of repeated code should not do much harm. Also using class method means every subclass will have to override the method, otherwise you end up with `Subclass.create() -> Superclass`